i18n/en/pop_desktop_widget.ftl

```diff
@@ -69,11 +69,11 @@ page-dock = Dock
 page-main = General
 page-workspaces = Workspaces
 
-position-bottom = Bottom of the screen
-position-left = Along the left side
-position-right = Along the right side
+position-bottom = Bottom
+position-left = Left
+position-right = Right
```

Comment:
I don't understand why the option for "bottom" uses a completely different sentence from the other two. In my opinion, there is no need to even be using sentences here, as it just makes it harder to read and understand the difference between the three options. By reducing each option to the only important word, it becomes much easier to read.